src/main/java/frc/robot/subsystems/CoralIntakeSubsystem.java

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import com.revrobotics.spark.SparkMax;
+import com.revrobotics.spark.SparkLowLevel.MotorType;
+
+public class CoralIntakeSubsystem extends SubsystemBase {
+  /** Creates a new CoralIntake. */
+  private final SparkMax coralIntakeMoter = new SparkMax(0, MotorType.kBrushless);
```

Comment:
Move ID logic from here -> constants file. Make it something other than 0 as a placeholder (e.g. 18)